powerauth-push-model/src/main/java/io/getlime/push/model/entity/PushMessageBody.java

```diff
@@ -177,4 +177,22 @@ public String getIcon() {
     public void setIcon(String icon) {
         this.icon = icon;
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
For easier comparison PushMessageBody objects used in tests when same method was called twice, so their references couldn't equal.